roles/convert2rhel/tasks/activation_keys.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: "Get organization (SCA) info"
+  theforeman.foreman.organization_info:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    name: "{{ foreman_organization }}"
+  register: org
```

Comment:
Ansible has this odd behavior, where it leaks variables/facts/anything after a role is done, so this `org` will be defined until the play has ended. Because of that, I'd prefer if we could use something longer, and less generic? Maybe `__foreman_convert2rhel_org_info` or something?